Gemfile

```diff
@@ -6,4 +6,4 @@ if facterversion = ENV['FACTER_GEM_VERSION']
   gem 'facter', facterversion, :require => false
 else
   gem 'facter', :require => false
-end
+end
```

Comment:
:-(

Reply:
I didn't want to get rubocop involved as it would have changed a whole lot of code.  But now that this is merged, let the games begin. Also we need to bump the version.

Reply:
rubocop++, and I would suggest the [PDK template](https://github.com/puppetlabs/pdk/blob/master/.rubocop.yml), obviously. But final say is with @mcanevet and @raphink . 

the version gets bumped during release-prep in accordance with the changes in the release.

Reply:
@DavidS I didn't have time to check PDK yet but feel free to add this in this module. Ping me onde it's OK for release.